azurelinuxagent/common/event.py

```diff
@@ -282,6 +281,51 @@ def _log_event(name, op, message, duration, is_success=True): # pylint: disable=
         logger.info(_EVENT_MSG, name, op, message, duration)
 
 
+class EventDebugInfo(object):
```

Comment:
"EventDebugInfo" seems to generic a name. Consider renaming or adding a doc string describing what this class is meant for

Reply:
Done